Resources/Maps/edge.yml

```diff
@@ -78300,8 +78214,8 @@ entities:
     - type: Transform
       pos: -42.5,19.5
       parent: 2
-      deviceLists:
-      - 11998
+    - type: DeviceNetwork
+      configurators:
```

Comment:
remove all these

Reply:
Only remove the configurator line. Type.... Honestly, just remap those (follow coordinates to find) they're scuffed for some reason